### PR TITLE
Add AssemblyScript for Arduino Blink

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

File: platforms/arduino_blink/wasm_assemblyscript/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "stripts": {
```
Comment (ReAlUA) [inline platforms/arduino_blink/wasm_assemblyscript/package.json]: `scripts`

Comment (MaxGraey) [reply platforms/arduino_blink/wasm_assemblyscript/package.json]: Good catch! Thanks!

File: platforms/arduino_blink/wasm_assemblyscript/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Compile
+asc app.ts -b app.wasm  \
```
Comment (vshymanskyy) [inline platforms/arduino_blink/wasm_assemblyscript/build.sh]: Use npx to run asc?

Comment (vshymanskyy) [inline platforms/arduino_blink/wasm_assemblyscript/build.sh]: Use npx to run asc?

File: platforms/arduino_blink/wasm_assemblyscript/package.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "scripts": {
+    "build": "npm asbuild:optimized && xxd -i app.wasm > app.wasm.h",
```
Comment (vshymanskyy) [inline platforms/arduino_blink/wasm_assemblyscript/package.json]: npm **run** asbuild:optimized ?

File: platforms/arduino_blink/wasm_assemblyscript/build.sh (hunk 2 above)
Comment (MaxGraey) [reply platforms/arduino_blink/wasm_assemblyscript/build.sh]: Done!

Comment (MaxGraey) [reply platforms/arduino_blink/wasm_assemblyscript/build.sh]: Done!

File: platforms/arduino_blink/wasm_assemblyscript/package.json (hunk 3 above)
Comment (MaxGraey) [reply platforms/arduino_blink/wasm_assemblyscript/package.json]: Done!

**Pull request merged**